Clarifications regarding "apu" and "apv" (#49)
ietf-wg-jose/draft-ietf-jose-hpke-encrypt: draft-ietf-jose-hpke-encrypt.md

HPKE has two places at which applications can specify auxiliary authenticated information as described in { Section 8.1 of RFC9180 }.

> Despite appearing similar to ECDH-ES, JOSE-HPKE does not use the "apu" and "apv" headers, described in Section 4.6.1 of {{RFC7518}}.

Reviewer: ```suggestion
Despite similarities to ECDH-ES, HPKE does not use the `apu` and `apv` header parameters, which are described in Section 4.6.1 of {{RFC7518}}.
```
